TrackingAntennaV3/main/Accelerometer.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file   Magnetometer.hpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ * @brief  
```

Comment:
Describe what this file is. From reading the header I should know how the accelormeter communicates with the arduino (SPI, I2C???), like why do you need the compass id? Is it beacuse its I2C. This is where you should also mention what sensor you're using, and in your case mention that the sensor also acts as a magnetometer.